client/networkmanager/Makefile

```
@@ -0,0 +1,80 @@
+
+# libnm.pc does not export a "plugindir" variable; the real plugin directory
+# is libdir/NetworkManager (e.g. /usr/lib64/NetworkManager on Fedora).
+PLUGINDIR ?= $(shell pkg-config --variable=plugindir libnm 2>/dev/null)
+ifeq ($(PLUGINDIR),)
+PLUGINDIR := $(shell pkg-config --variable=libdir libnm 2>/dev/null)/NetworkManager
+endif
+
+GOFILES := $(shell find . -name '*.go')
+
+.PHONY: all properties install uninstall clean
+
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)